server/model/monitor.js

```diff
@@ -980,7 +980,15 @@ class Monitor extends BeanModel {
             await R.store(bean);
 
             log.debug("monitor", `[${this.name}] prometheus.update`);
-            this.prometheus?.update(bean, tlsInfo);
+            let uptimeMetrics = {};
+            let data24h = await uptimeCalculator.get24Hour();
+            let data30d = await uptimeCalculator.get30Day();
+            let data1y = await uptimeCalculator.get1Year();
```

Comment:
What is the performance impact of this?
This makes doing a ping substancially more exspensive...
Unsure if this is an isssue.

Have you tested this?